app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -693,5 +698,21 @@ class BrowserTabViewModel(
     override fun externalAppLinkClicked(appLink: IntentType) {
         command.value = HandleExternalAppLink(appLink)
     }
+
+    override fun requiresAuthentication(request: BasicAuthenticationRequest) {
+        Timber.v("requiresAuthentication for [${request.site}]")
```

Comment:
Can we kill this comment? At first glance i thought this might log out credentials. I realise that's totally not the case, but even just to avoid anyone else having a similar thought, there's probably no need for the comment anyway.